libstorage/src/Volume.cc

```diff
@@ -2007,19 +2006,19 @@ int Volume::doCryptsetup()
 		replaceAltName( "/dev/dm-", Dm::dmDeviceName(mnr) );
 		}
 	    else
-	    {
+	        {
 		getMajorMinor( dmcrypt_dev, dummy, minor );
 		replaceAltName("/dev/dm-", Dm::dmDeviceName(minor));
-	    }
+	        }
 
 	    ProcPart p;
 	    unsigned long long sz;
 	    if( p.getSize( Dm::dmDeviceName(minor), sz ))
 		setSize( sz );
 	    }
-	}
+        }
     else
-	{
+        {
```

Comment:
Yes, indeed it is. ;-(

The first thing I did when I started working on libstorage last year in April was to fix that indentation. We actually did have a mandatory coding style for C/C++ even back then, but certain people chose to ignore that completely.